pkg/download/download.go

```diff
@@ -38,6 +39,18 @@ func DownloadBinary(unparsedURL string) (string, error) {
 		binaryDirectory = env.DefaultBinaryDir
 	}
 
+	dir, _ := filepath.Split(binaryDirectory)
```

Comment:
I just realized this could create problem if we use as Binary Directory "/home/foo" then this call would return dir = "/home" and we throw away the foo part. So this function call might be a bit of a foot gun and since we already have the binaryDirectory, we don't need to split it up anyway

Reply:
Sorry I missed this earlier.

Reply:
No if we want to use the binary directory "/home/foo/" then the foo part wouldn't be removed. This is the behavior we expect.